custom-namespaces.yaml

```diff
@@ -773,10 +773,14 @@ marketing:
   owners:
     - kignasiak@mozilla.com
   views:
-    acoustic_contacts:
+    acoustic_contact_change_events:
```

Comment:
Would it be better to create a new one and then remove the previous one, to avoid conflicts with a non existing namespace in Looker?

Reply:
This namespace is not really used yet, and I have another PR to add explores for those views which I have modified to compensate for this change. I just thought since usage is limited not then I might as well use the opportunity to make names a bit more explicit

Reply:
Sound good